src/core/electric_utility.jl

```diff
@@ -156,6 +156,11 @@ struct ElectricUtility
         outage_probabilities::Array{<:Real,1} = isempty(outage_durations) ? Float64[] : [1/length(outage_durations) for p_i in 1:length(outage_durations)],
         outage_time_steps::Union{Nothing, UnitRange} = isempty(outage_durations) ? nothing : 1:maximum(outage_durations),
         scenarios::Union{Nothing, UnitRange} = isempty(outage_durations) ? nothing : 1:length(outage_durations),
+
+        ### Grid Renewable Energy Fraction Inputs ###
+        # Utilities renewable energy fraction. Can be scalar or timeseries (aligned with time_steps_per_hour)
+        clean_energy_fraction_series::Union{Real, Array{<:Real, 1}} = Float64[],
```

Comment:
Can add both of these inputs to the markdown text at the top of the file (this then will appear in the documentation: https://nrel.github.io/REopt.jl/dev/)